src/components/board/Stack.vue

```diff
@@ -247,6 +250,13 @@ export default {
 		}
 	}
 
+	.stack--title {
+		white-space: nowrap;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		max-width: 200px;
```

Comment:
Please use calc to get the maximum width based on `$stack-width` so it continues to work if we change the global stack width at some point.